ci/obd-demo.yaml

```diff
@@ -144,7 +144,7 @@ metadata:
   annotations:
     kardinal.dev.service/dependencies: "productcatalogservice:http,cartservice:http"
     kardinal.dev.service/plugins: |
-      - name: https://github.com/kurtosis-tech/free-currency-api-plugin.git
+      - name: https://github.com/lostbean/free-currency-api-plugin.git
```

Comment:
You need to use this fork to validate the CI execution. Right?
If that is the case, don't forget to create another PR to revert these changes after the plugin PRs merge.

Reply:
yes, that was only to pass the CI tests in this branch. I'll merge the PRs on all plugins, revert and merge it.